### PR TITLE
Add geo commands

### LINKED ISSUE #5: Represent geo commands
https://redis.io/commands#geo

### PR DESCRIPTION
**Pull request by paulpdaniels**
Closes #5

### CODE REVIEW

File: src/main/scala/zio/redis/api/Geo.scala [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  lazy val geoDist =
+    Command(
+      "GEODIST",
+      Tuple4(
+        StringInput,
+        StringInput,
+        StringInput,
+        OptionalInput(GeoRadiusUnitInput)
+      ),
+      StringOutput
```
Comment (mijicd) [inline src/main/scala/zio/redis/api/Geo.scala]: I'd represent it as double, optional one according to the specs.

File: src/main/scala/zio/redis/api/Geo.scala [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    )
+
+  lazy val geoHash =
+    Command(
+      "GEOHASH",
+      NonEmptyList(StringInput),
```
Comment (mijicd) [inline src/main/scala/zio/redis/api/Geo.scala]: I'd keep key separate, e.g. `Tuple2(StringInput, NonEmptyList(StringInput))`.

File: src/main/scala/zio/redis/Input.scala [hunk 3]
```diff
@@ -52,5 +61,19 @@ object Input {
     _6: Input[F]
   ) extends Input[(A, B, C, D, E, F)]
 
+  final case class Tuple11[-A, -B, -C, -D, -E, -F, -G, -H, -I, -J, -K](
```
Comment (mijicd) [inline src/main/scala/zio/redis/Input.scala]: Make sure to add syntax for this to Command's companion. Also, please delete `Tuple6` as it doesn't seem to be used.

File: src/main/scala/zio/redis/api/Geo.scala (hunk 1 above)
Comment (paulpdaniels) [reply src/main/scala/zio/redis/api/Geo.scala]: We were missing both `OptionalOutput` and `DoubleOutput` so I added them, let me know if that is not what you meant.

File: src/main/scala/zio/redis/api/Geo.scala (hunk 2 above)
Comment (paulpdaniels) [reply src/main/scala/zio/redis/api/Geo.scala]: Done

File: src/main/scala/zio/redis/Input.scala (hunk 3 above)
Comment (paulpdaniels) [reply src/main/scala/zio/redis/Input.scala]: Done

**Comment by mijicd** [comment]
@paulpdaniels fix the imports, looks good otherwise!

**Pull request merged**